<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="418" started-at="2019-10-18T18:04:27Z" finished-at="2019-10-18T18:04:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="418" started-at="2019-10-18T18:04:27Z" finished-at="2019-10-18T18:04:28Z">
      <class name="com.test.excelWorking.Abc">
        <test-method status="FAIL" signature="testMethod(java.lang.String)[pri:0, instance:com.test.excelWorking.Abc@4b9e13df]" name="testMethod" duration-ms="394" started-at="2019-10-18T23:34:27Z" data-provider="test" finished-at="2019-10-18T23:34:28Z">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
Data provider mismatch
Method: testMethod([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)CompanyName,(java.lang.String)FirstName,(java.lang.String)LastName]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: testMethod([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)CompanyName,(java.lang.String)FirstName,(java.lang.String)LastName]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod -->
      </class> <!-- com.test.excelWorking.Abc -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
